DESCRIPTION.md

```diff
@@ -8,6 +8,10 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
+- v3.11.0(TBD)
+
+  - Added support for `token_file_path` connection parameter to read an OAuth token from a file when connecting to Snowflake.
```

Comment:
why we prefer file input rather than a string/byte input in this case?

Reply:
In SPCS, we tried to avoid that because the token file is generated by snowflake and refreshed every 10 minutes